README.md

```diff
@@ -1,2 +1,69 @@
-# conference-page
-This is a conference page
+
+# Project Name
+
+> WTA Summit: Women In Tech Africa Summit. This is Bi-yearly event that aims to inform, inspire and catalyze collective efforts to reduce the gender and digital gap in STEAM in Africa by 2035.
+>
+![Screenshot from 2022-05-21 01-26-56](https://user-images.githubusercontent.com/42035795/169627010-22ff7b8a-27cb-4f92-bee2-672fab40f05d.png)
+
+
+| *A Screenshot of the Desktop Version of the WTA Summit Website.*
+## Built With
+
+### Languages:
+- HTML
+- CSS
+- SCSS
+- JavaScript
+
+### Technologies used:
+-  _**GIT, GITHUB, LINTERS**_
+
+## Live Demo ()
+
+This is hosted on the live demo link below.
+
+[Live Demo Link](https://sirricelles.github.io/conference-page/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- Node.js
+- Node package manager(npm)
+### Setup
+- Verify you have node.js and npm installed
+- Clone the git repository
+### Install
+- run `npm install` to download all project dependencies
+### Usage
+- On the terminal or command prompt, run `npm run watch:sass`. This scripts compiles your sass code to css during development.
+- Server project pages via an Live server or any tool you prefer.
+
+## Authors
+
+👤 **Sirri Celles**
+
+- GitHub: [@SirriCelles](https://github.com/SirriCelles)
+- Twitter: [@SirriCelles](https://twitter.com/SirriCelles?t=fZl0blItFUQDC5vozH47nA&s=09)
+- LinkedIn: [Sirri Celles](https://www.linkedin.com/in/sirri-celles)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [ ] Kindly fix this broken link, this will make your readme more professional 👍🏿